api/src/main/java/com/kodemon/api/dto/UserDTO.java

```diff
@@ -28,7 +28,6 @@
     private Set<BadgeDTO> badges = new HashSet<>();
     @JsonProperty(required = false, defaultValue = "[]")
     private List<PokemonDTO> pokemons = new ArrayList<>();
-    @JsonProperty(required = false, defaultValue = "[]")
     private List<PokemonDTO> activePokemons = new ArrayList<>();
```

Comment:
Add `@JsonIgnore` annotation here instead. We do not want to send this to user via REST.